recipe/meta.yaml

```
@@ -79,7 +80,8 @@ outputs:
         - numpy
         - libtiledbvcf {{ version }}
         - python
-        - pyarrow
+        - pyarrow <13                            # [osx and py<312]
+        - pyarrow                                # [not osx and py>=312]
```

```suggestion
        - pyarrow                                # [not osx or py>=312]
```

https://en.wikipedia.org/wiki/De_Morgan%27s_laws